### PR TITLE
feat(core): add examples of enable and disable token

### PR DESCRIPTION
**Pull request by JoaquinTissera-Simtlix**
add examples of enable and disable token

STLX-13257

### CODE REVIEW

File: modules/core/example/js/algo/account-disable-token.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        {
+          amount: '0',
+          address: address,
+        },
+      ],
+      feeRate: 1,
+      walletPassphrase,
+      closeRemainderTo,
+    });
+    console.dir(txEnableToken, { depth: 6 });
```
Comment (JulioAchaval-SimTLiX) [inline modules/core/example/js/algo/account-disable-token.js:76]:
```suggestion
    const txDisableToken = yield wallet.sendMany({
      type: 'disabletoken',
      recipients: [
        {
          amount: '0',
          address: address,
        },
      ],
      feeRate: 1,
      walletPassphrase,
      closeRemainderTo,
    });
    console.dir(txDisableToken, { depth: 6 });
```

**Review by JulioAchaval-SimTLiX — CHANGES_REQUESTED** [review]

File: modules/core/example/ts/algo/disable-token.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ * And so
+ */
+
+import { BitGo, Wallet } from 'bitgo';
+import { WalletCoinSpecific } from '../../../src/v2/wallet';
+   
+// change this to env: 'production' when you are ready for production
+const bitgo = new BitGo({ env: 'test' });
+   
+// change this to 'algo' when you are ready for production
```
Comment (nicolasbellanich-simtlix) [inline modules/core/example/ts/algo/disable-token.ts:16]:
```suggestion
// change this to 'algo:16026733' when you are ready for production
```

File: modules/core/example/ts/algo/disable-token.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+// this can be retrieved by logging into app.bitgo-test.com (app.bitgo.com for production)
+// and going to: User > User Settings > Access Tokens > (+ icon)
+// the token will need Spender permission for ALGO
+const accessToken = 'insert access token string here';
+  
+const address = 'your root address or address of same wallet';
+
+const closeRemainderTo = 'your address where send token';
+  
+async function main() {
```
Comment (nicolasbellanich-simtlix) [inline modules/core/example/ts/algo/disable-token.ts:38]: Since it's TS , you should add return type
```suggestion
async function main() : void {
```

**Review by JulioAchaval-SimTLiX — DISMISSED** [review]

File: modules/core/example/ts/algo/disable-token.ts [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  
+  // your spendable balance will be the balance on the wallet root address
+  // and should differ from your confirmed balance
+  console.log('Spendable Balance:', wallet.spendableBalanceString());
+  
+  // we have to unlock this session since we're sending funds
+  const unlock = await bitgo.unlock({ otp, duration: 3600 });
+  if (!unlock) {
+    console.log('We did not unlock.');
+    throw new Error();
```
Comment (nicolasbellanich-simtlix) [inline modules/core/example/ts/algo/disable-token.ts:70]: Void error message it's maybe something we don't want to face in future
```suggestion
    throw new Error('Did not unlock');
```

File: modules/core/example/ts/algo/enable-token.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+ */
+
+
+import { BitGo, Wallet } from 'bitgo';
+import { WalletCoinSpecific } from '../../../src/v2/wallet';
+  
+// change this to env: 'production' when you are ready for production
+const bitgo = new BitGo({ env: 'test' });
+  
+// change this to 'algo' when you are ready for production
```
Comment (nicolasbellanich-simtlix) [inline modules/core/example/ts/algo/enable-token.ts:17]: Same as above

File: modules/core/example/ts/algo/enable-token.ts [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+ 
+  // your spendable balance will be the balance on the wallet root address
+  // and should differ from your confirmed balance
+  console.log('Spendable Balance:', wallet.spendableBalanceString());
+ 
+  // we have to unlock this session since we're sending funds
+  const unlock = await bitgo.unlock({ otp, duration: 3600 });
+  if (!unlock) {
+    console.log('We did not unlock.');
+    throw new Error();
```
Comment (nicolasbellanich-simtlix) [inline modules/core/example/ts/algo/enable-token.ts:70]: Same as above

File: modules/core/example/ts/algo/disable-token.ts (hunk 4 above)
Comment (JulioAchaval-SimTLiX) [inline modules/core/example/ts/algo/disable-token.ts:70]:
```suggestion
    throw new Error('We did not unlock.');
```

Comment (JulioAchaval-SimTLiX) [reply modules/core/example/ts/algo/disable-token.ts:70]: Try to avoid console.log when you throw an error

**Review by nicolasbellanich-simtlix — CHANGES_REQUESTED** [review]
Leave some comments here

**Review by nicolasbellanich-simtlix — DISMISSED** [review]

**Review by nicolasbellanich-simtlix — APPROVED** [review]
LGTM

**Review by JulioAchaval-SimTLiX — APPROVED** [review]

**Review by lcovar — APPROVED** [review]

**Pull request merged**